cumulus/primitives/core/src/lib.rs

```diff
@@ -78,6 +78,43 @@ impl From<MessageSendError> for &'static str {
 	}
 }
 
+/// The origin of an inbound message.
+#[derive(Encode, Decode, MaxEncodedLen, Clone, Eq, PartialEq, TypeInfo, Debug)]
+pub enum AggregateMessageOrigin {
+	/// The message came from the para-chain itself.
```

Comment:
is it "para-chain" or "parachain"?

Reply:
I personally write `para-chain` since it complements `relay-chain` better than one-worders like `relaychain`.